docs/docs/domain.mdx

```diff
@@ -60,14 +56,14 @@ intents:
 
 This means that excluded entities for those intents will be unfeaturized and therefore
 will not impact the next action predictions. This is useful when you have
-an intent where you don't care about the entities being picked up. 
+an intent where you don't care about the entities being picked up.
 
 If you list your intents without this parameter, the entities will be
 featurized as normal.
 
 :::note
 If you really want these entities not to influence action prediction we
-suggest you make the slots with the same name of type `unfeaturized`.
+suggest you make the slots with the same name of type `any`.
```

Comment:
Maybe instead we should recommend adding the `influence_conversation: false`? It's probably more self explanatory. And we should link to the section